docs/reference_dataset/template_repository/hello-world-root-depls/cf-apps-deployments/generic-app/template/generic-app_manifest-tpl.yml

```diff
@@ -2,3 +2,19 @@ name: generic-app
 instances: (( grab secrets.generic-app.instances ))
 path: .
 
+memory: (( grab secrets.undefined_value || undefined.fallback.value || "64M" )) # as no value are defined, spruce will use the 64M default value
+
+# By default, spruce home dir is deployment directory (ie: credentials-resource/hello-world-root-depls/cf-apps-deployments/generic-app/)
+disk_quota: (( file "spruce-file-sample-from-secrets.txt" ))
+
+# To access generated files: "../../../../generated-files"
+# To access paas-templates directory, it is possible to use one of the following:
+#   - direct reference, quite similar, like  "../../../../additional-resource/hello-world-root-depls/cf-apps-deployments/generic-app/template"
+#   - use a variable see sample with timeout and timeout_from_file $CUSTOM_SCRIPT_DIR: (( grab $CUSTOM_SCRIPT_DIR )), in this sample, CUSTOM_SCRIPT_DIR=additional-resource/hello-world-root-depls/cf-apps-deployments/generic-app/template
+secrets: # this key is pruned by spruce
+  timeout_from_file: ((concat "../../../../" $CUSTOM_SCRIPT_DIR "/spruce-file-sample-from-templates.txt" ))
+
+timeout: (( file secrets.timeout_from_file ))
```

Comment:
`timeout: (( file secrets.path_to_my_file ))`

Reply:
@gberche-orange, instead of keeping it undocumented, I'd like to add a warning reminder, so user know it exists and can comments the related issue if needed.

`#### Below a NOT RECOMMENDED and subject to change option. See https://github.com/orange-cloudfoundry/cf-ops-automation/issues/259
`  

What do you think ?

Reply:
sounds good. thanks